dts/bindings/opamp/st,stm32-opamp.yaml

```
@@ -0,0 +1,11 @@
+# Copyright (c) 2025 Alexander Kozhinov <ak.alexander.kozhinov@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+title: STM32 OPAMP (Operational Amplifier) device tree bindings.
+
+description: STM32 OPAMP (Operational Amplifier)
```

`title` is optional and not necessary when the description is short enough
```suggestion
description: STM32 OPAMP (Operational Amplifier)
```